Dockerfile

```diff
@@ -19,7 +19,8 @@ RUN wget -qO- https://surfer.nmr.mgh.harvard.edu/pub/dist/freesurfer/6.0.1/frees
 
 RUN apt-get install -y python3
 RUN apt-get install -y python3-pip
-RUN pip3 install nibabel pandas
+RUN apt-get install -y python3-pandas
```

Comment:
What's the error that caused you to add this? That looks like it would be some trouble if you have pandas installed in different site package directories...

Are python 2 and 3 needed for correct execution? It would slim the image a bit to remove 2.

Reply:
I'm with @leej3 on this one! Given the recent developments we should only support 3 assuming that we don't need 2 for some steps which should then be adapted.

Reply:
I agree, that this is a dirty, but it was a quick hack to get things working. If it's alright, I'm not going to fix this since we'll be moving to neurodocker shortly.